sphinx/source/conf.py

```diff
@@ -23,18 +23,18 @@
 # -- Project information -----------------------------------------------------
 
 project = config['sphinx']['name']
+package_dir = config["sphinx"]["package_dir"]
 author = config['metadata']['author']
 copyright = str(date.today().year) + ', ' + author
-package_dir = config["sphinx"]["package_dir"]
 
-sys.path.insert(0, os.path.join(BASE_FOLDER, package_dir))
+sys.path.insert(0, os.path.join(BASE_FOLDER))
 
 # -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.autodoc', 'm2r2']
+extensions = ['sphinx.ext.autodoc', 'm2r2', 'sphinx.ext.githubpages']
```

Comment:
This further extension allows to automatically add `.nojekyll` empty file to `docs/html` while running `make --directory=sphinx --file=Makefile clean html`